analyses/mwi/dryspells_trigger/exploration/2021_2022_postseason_data_analysis.R

```diff
@@ -0,0 +1,148 @@
+# analyse precipitation data for postseason analysis of 2021-2022 season
+# this script sources 2021_2022_postseason_data_pull.R
+# this script is sourced by 2021_2022_postseason_overview.Rmd
+
+#####
+## data pull
+#####
+
+# load data and useful R objects
+source('dryspells_trigger/exploration/2021_2022_postseason_data_pull.R')
+
+#####
+## data formatting
+#####
+
+# user-friendly date labels
+dates <- data.frame(dates = seq(from = as.Date('2021-10-01', format = '%Y-%m-%d'), 
```

Comment:
Easier if you move this could into the `data_pull` file because then it's where you wrangle the other dates and you can just very easily use the numeric vector there directly. I've put code for that in that review.